notebooks/samples/explanations/tf2/ai-explanations-image.ipynb

```diff
@@ -95,7 +95,7 @@
       "source": [
```

Comment:
<p>Correction in <code>VALIDATION_SPLIT</code>: ..."after it is trained"</p><p> </p><p>General comment: Whenever you've described what parameters and methods do, start off the description with the verb.</p><p> </p><p>For example, in your parameter / code descriptions here you'd remove "This is" / "This" and just say:</p><p> </p><ul><li>Cloud Storage location for...</li><li>Sets aside 20% of the dataset as...</li><li>Mapping of...</li><li>Gets a list of...</li><li>Randomly shuffles...</li><li>Calculates...</li><li>Calculates...</li></ul><p> </p><p>I've provided multiple other examples of this in subsequent review comments on this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>